lab-yunjoo/server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
Don't forget to add comments to help people reading your code, and for yourself.